### PR TITLE
move k8s specific params to params_k8s.go

### PR DESCRIPTION
**Pull request by tariq1890**
Moving Kubernetes parameters to its rightful place :).

cc: @jsturtevant

### CODE REVIEW

File: pkg/acsengine/params_k8s.go [hunk 1]
```diff
@@ -299,6 +302,23 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
...
 			addValue(parametersMap, "enableAggregatedAPIs", kubernetesConfig.EnableAggregatedAPIs)
+
+			if properties.HasWindows() {
+				// Kubernetes packages as zip file as created by scripts/build-windows-k8s.sh
+				// will be removed in future release as if gets phased out (https://github.com/Azure/acs-engine/issues/3851)
+				kubeBinariesSASURL := kubernetesConfig.CustomWindowsPackageURL
+				if kubeBinariesSASURL == "" {
+					kubeBinariesSASURL = cloudSpecConfig.KubernetesSpecConfig.KubeBinariesSASURLBase + k8sComponents["windowszip"]
+				}
+				addValue(parametersMap, "kubeBinariesSASURL", kubeBinariesSASURL)
```
Comment (jackfrancis) [inline pkg/acsengine/params_k8s.go]: Let's also move these 4 params definitions out of `kubernetesbase.t` and into `windowsparams.t`

Comment (tariq1890) [reply pkg/acsengine/params_k8s.go]: Sure!

**Comment by jsturtevant** [comment]
looks good :+1:  I'll review again after you move the params out of `kubernetesbase.t`

**Pull request merged**